### ISSUE TITLE
Add an HTML linter to the tests

### ISSUE THREAD

**Issue by mgwalker** [18F/dol-whd-14c#230]
Having an HTML linter as part of the tests would help keep the HTML consistent (e.g., consistent indentation, properly closed tags, etc.).  It'd be a nice bonus if the linter could just fix any indentation or other whitespace inconsistencies, so the dev team doesn't have to spend time thinking about it.

**Comment by mmurthydol** [comment]
@mgwalker Thought you finished this story Greg?

**Comment by mgwalker** [comment]
Nope.  There's a JS linter, but not an HTML one.  I'm not 100% sure that an HTML linter would work correctly with Angular, but it's still work checking out.

**Comment by mmurthydol** [comment]
@phirefly @EStriegel Icebox. This is caviar for the homeless.
